### PR TITLE
Adding state management

### LINKED ISSUE #35: Set up state management
**User Story:**

As a developer, I want to have a manager for states, so that I can easily access another component's state

**Acceptance Criteria:**

Set up store so that it can be access by all components
Related components are updated when store is updated

**Additional Context:**

### PR DESCRIPTION
**Pull request by Sabatr**
## Link to Issue Number
Closes #35 

## Description
Used Redux for state management between components
Set up a basic example using the number of seats 

## Checklist
- [x] All feature request A/C have been met OR the bug has been fixed
- [ ] I ran all necessary tests and they pass
- [x] I rebased upstream/master onto my working branch before opening this PR
- [x] I have named my PR something sensible
- [x] I have included the issue number above
- [x] I have assigned atleast two people to review my changes

## Other Notes
Other notes/context

### CODE REVIEW

File: frontend/src/store/booking/bookingReducer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { actionType } from "./bookingActions";
+
+const initialState = {
+  numberOfSeats: null,
+};
+
+const bookingReducer = (state, action) => {
```
Comment (nidhineshnand) [inline frontend/src/store/booking/bookingReducer.js]: Could comments be potentially helpful here

Comment (Sabatr) [reply frontend/src/store/booking/bookingReducer.js]: Hmm for what?

Comment (nidhineshnand) [reply frontend/src/store/booking/bookingReducer.js]: To explain what booking reducer does or maybe I'm just confused

Comment (Sabatr) [reply frontend/src/store/booking/bookingReducer.js]: Done

Comment (Sabatr) [reply frontend/src/store/booking/bookingReducer.js]: Please merge when you can @nidhineshnand

**Pull request merged**